linux-patches/README.md

```diff
@@ -1,7 +1,8 @@
 # Linux patches
 
 These are patches that are in the process of being upstreamed but needs to
-be patched now to support some critical functionallity.
+be patched now to support some critical functionallity. They will be
+applied automatically when the Linux kernel tar file is extracted.
```

Comment:
It's not like they were documented before, but if you want feel free to mention menuconfig / saveconfig flow here.

Reply:
The top level `README.md` could use an overhaul to make it clearer how to build and configure for different platforms.  Updating the `$PLATFORM` variable in the makefile is the same hack we're doing in Heads and I wish there were a better way (like Kconfig, if it didn't have so much overhead to integrate)